clients/snowflake/snowflake_test.go

```diff
@@ -211,7 +211,7 @@ func (s *SnowflakeTestSuite) TestExecuteMergeDeletionFlagRemoval() {
 	rowsData := make(map[string]map[string]any)
 	for i := 0; i < 5; i++ {
 		rowsData[fmt.Sprintf("pk-%d", i)] = map[string]any{
-			"id":         fmt.Sprintf("pk-%d", i),
```

Comment:
This was pretty funny. We labeled `id` as an integer, but the values were `pk-1`, `pk-2`, etc.